cmd/cronjob_receive_adapter/main.go

```diff
@@ -69,7 +56,8 @@ const (
 func main() {
 	flag.Parse()
 
-	ctx := context.Background()
+	ctx := signals.NewContext()
+
 	var env envConfig
 	err := envconfig.Process("", &env)
 	if err != nil {
```

Comment:
Convert this to the 3-line version?

```golang
if err := envconfig.Process("", &env); err != nil {
  log.Fatalf(...)
}
```

(I also don't think panic's stack trace will help here.)

Reply:
done!